File: Java/6.23.java
/*
One lap around a standard high-school running track is exactly 0.25 miles. Write a program that takes a number of miles as input, and outputs the number of laps.

Output each floating-point value with two digits after the decimal point, which can be achieved as follows:
System.out.printf("%.2f", yourValue);

Ex: If the input is:

1.5
the output is:

6.00
Ex: If the input is:

2.2
the output is:

8.80
*/

import java.util.Scanner;

public class LabProgram {
   
   /* Define your method here */ 
   public static double milesToLaps(double userMiles) {
      final double MILES_PER_LAP = 0.25;
      return userMiles / MILES_PER_LAP;
   }
   
   public static void main(String[] args) {
      /* Type your code here. Your code must call the method.  */
      Scanner scan = new Scanner(System.in);
      double miles = scan.nextDouble();
      double laps = milesToLaps(miles);
      
      System.out.printf("%.2f", laps);
   }
}

File: CPP/6.23.cpp
/*
Write a program whose input is two integers and whose output is the two integers swapped.

Ex: If the input is:

3 8
the output is:

8 3
Your program must define and call a function. SwapValues returns the two values in swapped order.
void SwapValues(int& userVal1, int& userVal2)
*/

#include <iostream>
using namespace std;

/* Define your function here */ 
void SwapValues(int& userVal1, int& userVal2) {
   int tempVal = userVal1;
   userVal1 = userVal2;
   userVal2 = tempVal;
}

int main() {
   /* Type your code here. Your code must call the function.  */
   int userVal1, userVal2;
   
   cin >> userVal1 >> userVal2;
   
   SwapValues(userVal1, userVal2);
   
   printf("%d %d\n", userVal1, userVal2);

   return 0;
}
